### PR TITLE
More bots

### PR DESCRIPTION
**Pull request by portalvasco**
More bots added

### CODE REVIEW

File: bots.php [hunk 1]
```diff
@@ -70,8 +70,10 @@ function useronline_get_bots() {
 		'Radian6' => 'radian6',
 		'RadioUserland' => 'userland',
 		'Moz' => 'rogerbot',
+                'Rogerbot' => 'rogerbot',
```
Comment (lesterchan) [inline bots.php]: Can we fix the alignment?

Comment (portalvasco) [reply bots.php]: Yes of course. You can do that. Thanks.

Comment (lesterchan) [reply bots.php]: Can uyoufixed it in the PR?

Comment (portalvasco) [reply bots.php]: Done!

Comment (lesterchan) [reply bots.php]: You remove the bot rather than fix the spacing?

Comment (portalvasco) [reply bots.php]: Yes, i remove the bot because i see that it is listed twice.

**Pull request merged**